src/software/util/time/duration.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "software/util/time/duration.h"
 
+#include <cmath>
+
```

Comment:
doesn't seem within scope...

Reply:
Similarly, this failed when we changed compilers, I can only assume because compilation order shifted somewhat.